aikau/src/main/resources/alfresco/renderers/Indicators.js

```diff
@@ -81,6 +81,15 @@ define(["dojo/_base/declare",
       propertyToRender: null,
 
       /**
+       * Whether this indicator is clickable and publishes a action request or not.
+       *
+       * @instance
+       * @type {boolean}
+       * @default
+       */
```

Comment:
When we add a new configuration attribute or function we always add the @since tag with the version it was added in